enum/enum/ViewController.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet var hourImageView: UIImageView!
+    @IBOutlet var menuButton: UIButton! {
+        didSet {
+            //メニューの要素を追加
+            var menuItems = [UIAction]()
+            //選択によって画像を変更
+            for hour in Hours.allCases {
+                menuItems.append(UIAction(title: hour.title, image: hour.image) {[weak self] _ in
```

Comment:
ものすっごく細かくて申し訳ないんだけど、中括弧の前後はスペースを開けるという慣習があるので修正お願いします。🙏
```suggestion
                menuItems.append(UIAction(title: hour.title, image: hour.image) { [weak self] _ in
```